### DISCUSSION TITLE
Using the Yalmip model to solve it in C++

### DISCUSSION THREAD

**Discussion by ZhYGu** [yalmip/YALMIP#1111] [General discussion] [answered]
Hello Yalmip group, I am now encounter a problem that I want to use the Yalmip to model my MPC controller. However, I want to avoid using for loop in MATLAB. Is there anyway I can get the Yalmip model in gurobi form from matlab, and implement it in C/C++? Thanks.

**Comment by johanlofberg** [comment, marked as answer]
Avoiding for-loop, and implementing in C/C++ are two completely different things

If you by avoiding for-loop actually mean that you want to speed things up and avoid redefining from scratch after making small changes, you might want to implement it using an optimizer 

To extract data, https://yalmip.github.io/Extracting
